tutorial/README.rst

```diff
@@ -0,0 +1,5 @@
+Tutorial
+========
+
+.. toctree::
+   :maxdepth: 2
```

Comment:
The table of content employs the grid layout, and it may be less obvious which one is the starting point of the tutorial than the previous text list. So, how about adding a short description to guide users?
(c.f., [PyTorch tutorial](https://pytorch.org/tutorials/) also has a headline to navigate new users.)
![image](https://user-images.githubusercontent.com/3255979/88635596-ce4c6e80-d0f2-11ea-96b6-0b8feee7199e.png)

Reply:
Resolved by https://github.com/optuna/optuna/pull/1543/commits/fd066f66714dcdddab5f41e7c13a40dc639e4910.